control_freq: 10.0
# X
kp_x: 0.15
ki_x: 0.03
kd_x: 0.015
# Y
kp_y: 0.15
ki_y: 0.06
kd_y: 0.015
# Z
kp_z: 0.2
ki_z: 0.005
kd_z: 0.0
max_vx: 0.15
max_vy: 0.15
max_wz: 0.15
resize_debug_img: 1.0